salt/states/archive.py

```diff
@@ -1350,10 +1350,13 @@ def extracted(name,
                         )
                         return ret
 
-                    tar_opts = shlex.split(options)
+                    # Ignore verbose file list options as we are already using
+                    # "v" below in tar_shortopts
+                    tar_opts = [x for x in shlex.split(options)
+                                if x not in ('v', '-v', '--verbose')]
```

Comment:
What happens if someone passes `xv`? Should this check be something like:

```
if not x.startswith('-') and 'v' not in x
```

in addition to what it is checking now?